### ISSUE TITLE
Connecting remotely

### ISSUE THREAD

**Issue by Ezraiel92** [s3condAtall/Deezloader#4]
Has anyone else been able to connect remotely? For example, I'm not able to connect to my vps in NC. I can get it to work locally.

**Comment by s3condAtall** [comment]
Did you checked the port number?

**Comment by Ezraiel92** [comment]
Yeah, I figured it out. However It crashes after awhile. Can't keep it up overnight etc.
